### PR TITLE
Adding Readzi application to showcase.

### PR DESCRIPTION
**Pull request by strazi**
Adding my application Readzi to the showcase.

### CODE REVIEW

**Comment by facebook-github-bot-2** [comment]
By analyzing the blame information on this pull request, we identified @brentvatne and @manonthemat to be potential reviewers.

**Comment by facebook-github-bot-6** [comment]
@strazi updated the pull request.

**Comment by facebook-github-bot-1** [comment]
@strazi updated the pull request.

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -749,6 +749,15 @@ var apps = [
     ],
   },
   {
+    name: 'Readzi',
+    icon: 'http://a3.mzstatic.com/us/r30/Purple18/v4/b9/32/99/b9329992-1677-9ee2-5d04-e901e4bbb2b7/icon175x175.png',
+    link: 'https://readzi.io',
+    author: 'Kevin Kennedy',
+    blogs: [
+      'https://strazi.org/journal/building-readzi/'
```
Comment (janicduplessis) [inline website/src/react-native/showcase.js]: Can you add trailing commas here and the line after.

**Comment by facebook-github-bot-2** [comment]
@strazi updated the pull request.

**Comment by janicduplessis** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-8** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1149792451738572/int_phab) to review.

**Pull request closed without merging**